docs/docs/user-guide/appendix/FAQ.md

```diff
@@ -0,0 +1,40 @@
+# Frequently Asked Questions
+
+### Is BioNeMo free to use?
+
+Yes, BioNeMo is free to use.
+
+### How do I install BioNeMo?
+
+BioNeMo is distributed as a Docker container through NVIDIA NGC. To download the pre-built Docker container and data
+assets, you will need a free NVIDIA NGC account.
+
+Alternatively, you can install individual sub-packages from within BioNeMo by following the corresponding README pages
+the [BioNeMo GitHub](https://github.com/NVIDIA/bionemo-framework).
+
+### How do I update BioNeMo to the latest version?
+
+To update the BioNeMo Docker container, you need to pull the latest version of the Docker image using the command
+`docker pull`. For available tags, refer to the
+[BioNeMo Framework page in the NGC Catalog](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/clara/containers/bionemo-framework).
+
+### What are the system requirements for BioNeMo?
+
+Generally, BioNeMo should run on any NVIDIA GPU with Compute Capability ≥8.0. For a full list of supported hardware,
+refer to the [Hardware and Software Prerequisites](../getting-started/pre-reqs.md).
+
+### Can I contribute code or models to BioNeMo?
+
+Yes, BioNeMo is open source and we welcome contributions from organizations and individuals.
```

Comment:
Nit: could mention or link to the contributing guide here too!

Reply:
Added!